features/admin.core.v1/providers/common-feature-provider.tsx

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
Add this code directly to app.tsx in console

Reply:
https://github.com/wso2/identity-apps/pull/5896/commits/e701ae49dfeb1db78f557d25afbd37f0e08e5f4a